Start-Up.md

```diff
@@ -0,0 +1,27 @@
+# Start-Up Procedure
+
+A quick-start procedure for starting up the AV-42C.
+
+<!-- TODO: put a picture here with a general layout and a note that it is subject to change -->
+
+On the right panel:
+
+1. Flip "Main Battery" switch to "On"
+2. Flip "Brake Lock" switch to "On" (this is so you don't accidentally roll while starting up)
+3. Flip "APU" switch to "On" (it is a covered switch, so you need to lift the cover first)
+4. Wait for the RPM gauge under the "APU" switch to spool up
+5. Flip the covered switch for one of the engines to "On" (you can try both at once, but you are likely to have start failures)
+6. Wait for the engine to spool up (RPM gauges for the engines are on the front panel on the right side)
+7. Flip the covered switch for the other engine to "On" and wait for it to spool up
```

Comment:
Suggestion: "Repeat steps 5. and 6. for the other engine"

Also, perhaps add a step (or just a note) to close the covered switch after it's spooled.

Reply:
> 7. Repeat steps 5 & 6 for the other engine, and close the covers.

What do you think of this for a replacement for 7 ?

Reply:
Sounds good to me 👍